fun/starboard v2/starboardListener.go.tmpl

```diff
@@ -0,0 +1,159 @@
+{{/*
+	starboard listener
+
+	This command allows users to react to messages within the starboard channel with stars/anti-stars. 
+
+	Recommended trigger: Reaction trigger with option `Added + Removed reactions`.
+	Channel limitations: run ONLY in your starboard channel.
+*/}}
+
+{{/* CONFIGURATION VALUES START */}}
+{{$starEmoji := "⭐"}} {{/* Star emoji name */}}
+{{$starEmFull := "⭐"}} {{/*full emoji name of star emoji, needed to add/remove reactions. Same as $starEmoji if unicode*/}} 
+{{$starLimit := 4}} {{/* Reactions needed for message to be put on starboard */}}
+{{$starboard := 712681639410663516}} {{/* ID of starboard channel */}}
+{{$maxAge := "2w"}} {{/*maximum age of message for stars to be counted. structure (mo)nth, (w)eek, (d)ay, (h)our. ex: 3d = 3days, 1mo = 1 month.*/}}
+{{$selfStarEnable := false}} {{/*true to allow members to star/anti-star their own posts, false to disallow*/}}
+{{$warnMessages := true}} {{/*true to enable sending of messages such as 'you can't star your own post', false to disable*/}}
+
+{{$antiStarEnable := false}} {{/*enable/disable anti-star counting. enable = true, disable = false*/}}
+{{$antiStarEmoji := "whystar"}} {{/*anti-star emoji name*/}}
+{{$antiStarEmFull := ":whystar:592143371565400064"}} {{/*full emoji name of anti-star emoji, needed to add/remove message reactions. same as $antiStarEmoji if unicode*/}}
```

Comment:
```suggestion
{{$antiStarEmoji := "❌"}} {{/*anti-star emoji name*/}}
{{$antiStarEmFull := "❌"}} {{/*full emoji name of anti-star emoji, needed to add/remove message reactions. same as $antiStarEmoji if unicode*/}}
```

Reply:
I intentionally left this one to display the difference in required syntax between the two variables if someone is using custom emotes. Happy to change to unicode X but unsure if it's likely to cause confusion, I can see someone who isn't too well versed trying to use something like "whystar" in both of them and then getting lost about why it's not working. Whichever is better in your professional opinion

Reply:
Would prefer default emojis, some just skim over it without paying proper intention, because they "just want to add it" - if people want to change it, they'll read it eventually.

Reply:
fair enough but shame on you for thinking people will ever do the "reading" thing :kek: